packages/slice-machine/components/AppLayout/Navigation/index.test.tsx

```diff
@@ -60,10 +60,17 @@ function renderApp({ canUpdate }: { canUpdate: boolean }): RenderReturnType {
           apiEndpoint: "https://foo.cdn.prismic.io/api/v2",
         },
         changelog: {
-          currentVersion: "",
-          updateAvailable: canUpdate,
-          latestNonBreakingVersion: null,
-          versions: [],
+          sliceMachine: {
+            currentVersion: "",
+            updateAvailable: canUpdate,
+            latestNonBreakingVersion: null,
+            versions: [],
+          },
+          adapter: {
```

Comment:
Should the `adapter` object contain the same properties as `sliceMachine`?

Reply:
At the moment, we don't need the same properties. I decided to only save in the store what is required. I don't know how we want to handle the changelog page in the future, so I kept it short in purpose. It would mean to save unused value in store, and it could be hard to debug and understand later on, WDYT?

Reply:
Makes sense! 🙂